### DISCUSSION TITLE
简介：PCL 功能投票

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#2] [功能投票]
你可以点击 `↑` 按钮进行投票，开发者会优先处理票数较高的帖子，但也注意这并不代表票数最高的一定就会被立即处理……只是优先级会更高一点……
如果一个功能已经被选中开始制作，它在 Discussions 中的对应帖子将被删除，对应的 Issue 将被标记为处理中。

### 点击 [这个链接](https://github.com/Hex-Dragon/PCL2/discussions/categories/%E5%8A%9F%E8%83%BD%E6%8A%95%E7%A5%A8) 查看所有进行中的功能投票项目。

![image](https://user-images.githubusercontent.com/40348278/192096613-83b55a4b-d2c6-4ce4-bb77-7b112459f167.png)

**Comment by xiaoouyanplayminecraft** [comment]
希望可以添加一个可以下载材质包的功能

**Reply by kunkun8888666** [reply]
还有光影

**Reply by iPodtouch6557** [reply]
这里不用于给你们提建议

**Comment by iPodtouch6557** [comment]
新功能label已经改了，要不要……重新截一下？

**Comment by AKing1328** [comment]
添加neoforge的下载

**Reply by iPodtouch6557** [reply]
这里不是提建议的地方………………………………………………
见 #2394 #2244 #2446 #2445 #2411 #2388 #2378 #2376 #2340 #2449 #2365 这些issues和discussions

**Comment by adminbbbbben** [comment]
增加自动检测更新

**Reply by zkitefly** [reply]
自动检测更新啊，还有发个 issue 吧

**Comment by kunkun8888666** [comment]
希望增加简便的连接功能

**Reply by iPodtouch6557** [reply]
这里不是你提建议的地方
